config/locales/en.yml

```diff
@@ -97,6 +97,8 @@ en:
               version.
             locations_cant_be_urls: App locations must not be URLs for this framework
               version.
+            invalid_v1_location: "%{invalid_locations} is an invalid location in framework
```

Comment:
need to support one or many? https://github.com/zendesk/zendesk_apps_support/blob/e16fdfb6fe5fa4111845db82d27c5fd28d6ade1d/config/locales/en.yml#L10-L11